docs/tutorials/plots.rst

```diff
@@ -133,7 +133,15 @@ time and issue the plotting command.
 
     observe_time = Time('2000-06-15 23:30:00')
 
-    plot_airmass(target, observer, observe_time)
+
+    # Use context manager to avoid earth orientation warning
```

Comment:
We need to find a better way to catch these, it should not be in the user facing docs pages at least (or not in all of them). I'll try to cook up comething.